### DISCUSSION TITLE
Failed to connect via relay server

### DISCUSSION THREAD

**Discussion by ddjanic** [rustdesk/rustdesk#5287] [General]
### Bug Description

Good afternoon, at yesterday - stopped connecting to  RustDesk network through some providers.
"Connection ErrorFailed to connect via relay server: Failed to connect to relay server: Please try later"

But if run VPN connect (OpenVPN, Ouline, etc.) and restart the service, it connects OK without problems.

I attached file C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\log\server\RustDesk_rCURRENT.log
[RustDesk_rCURRENT.log](https://github.com/rustdesk/rustdesk/files/12282044/RustDesk_rCURRENT.log)

### How to Reproduce

Russia, Rostelecom provider.

### Expected Behavior

Connect to relay.

### Operating system(s) on local side and remote side

Windows 10 x64 for Workstations

### RustDesk Version(s) on local side and remote side

1.2.1

### Screenshots

![2023-08-07_18-29-13](https://github.com/rustdesk/rustdesk/assets/1620653/7162dc70-2737-46c6-a84b-236c4be98035)

**Comment by rustdesk** [comment]
we have no way to solve your network problem.

**Reply by rakleed** [reply]
@rustdesk @21pages what are the IP addresses of public free servers? ISP support asks for diagnostics.

**Reply by shvchk** [reply]
@rakleed If you are from Russia, this post might be useful: https://4pda.to/forum/index.php?showtopic=1047138&view=findpost&p=125226737

**Reply by OptikITLik** [reply]
Looks like we need public servers in Russia
How can i contribute?

**Reply by rakleed** [reply]
@OptikITLik you should read discussions https://github.com/rustdesk/rustdesk/discussions/1026 and https://github.com/rustdesk/rustdesk/discussions/1657 first, and if your servers meet the conditions, you can create a [new discussion](https://github.com/rustdesk/rustdesk/discussions/new/choose) so that the owners of the repository will notice it.

**Reply by alexandr19972011** [reply]
> Looks like we need public servers in Russia How can i contribute?

@OptikITLik Hello! Is there any news regarding the creation of public servers in Russia? Roskomnadzor, as a state regulator, continues to block a wide variety of services and apparently has no plans to lift them. Unfortunately, not everyone has the ability to create their own server. I don't know how many people use it RustDesk in Russia, but a move like creating public servers would definitely benefit all Russian users. The problem is that the blocks also affect VPN-services, making it increasingly difficult to bypass these blocks.

Best wishes.

**Comment by xlionjuan** [comment]
Host your own RustDesk server

**Reply by ddjanic** [reply]
already up srv id && relay

**Comment by boingo-00** [comment]
My wild guess is that VPN blocking solutions break it
Can Rustdesk traffic be detected as a tunnel?
